ci/build.py

```diff
@@ -200,7 +200,7 @@ def default_ccache_dir() -> str:
         ccache_dir = "/tmp/_mxnet_ccache"
```

Comment:
Why wouldn't we do that on OSX as well?

Reply:
I was going to look at this, but the pr was merged. We can change it in another PR but have to test locally that docker can access the home folder.